travis-ci.sh

```diff
@@ -20,7 +20,9 @@ if [ ${BUILD_EXAMPLE=1} -eq 1 ]; then
 fi
 if [ ${RUN_TEST=1} -eq 1 ]; then
     for ex in `\ls -1 tests/`; do
-        echo "[INFO] Running test $ex"
-        (cd tests/$ex && dub --compiler=$DC && dub clean)
+        if [ -r test/$ex/dub.{json,sdl} ]; then
```

Comment:
Bash complains for Travis-CI:
> '[' -r test/vibe.core.net.1376/dub.json test/vibe.core.net.1376/dub.sdl ']'
> ./travis-ci.sh: line 23: [: test/vibe.core.net.1376/dub.json: binary operator expected

Should probably use `[ -r test/$ex/dub.json} ] || [ -r test/$ex/dub.sdl ]`.